### ISSUE TITLE
Android Flip Available for Android Studio?

### ISSUE THREAD

**Issue by selvamurali** [openaphid/android-flip#146]
Dear team
any have this Openaphid Android Flip for Android Studio, if anyone have kindly share here
Thank you

**Comment by gkarunakaran** [comment]
Yes you can use this same library in android studio also by importing as a module, for clear steps how to import this library in android studio follow this link [](http://www.theappguruz.com/blog/library-project-module-as-dependency-in-android-studio).

**Comment by anvesh523** [comment]
Try with this, https://github.com/anvesh523/android-flip/tree/master/FlipView/Demo

**Comment by fikr4n** [comment]
Hi, I also tried to convert it to Android Studio project: https://github.com/fikr4n/android-flip
